src/amo/api/reviews.js

```diff
@@ -262,3 +263,28 @@ export const flagReview = ({
     );
   });
 };
+
+type DeleteReviewParams = {|
+  apiState: ApiState,
+  errorHandler: ErrorHandlerType,
+  reviewId: number,
+|};
+
+export const deleteReview = ({
+  apiState,
+  errorHandler,
+  reviewId,
+}: DeleteReviewParams = {}): Promise<void> => {
+  return new Promise((resolve) => {
+    invariant(reviewId, 'The reviewId parameter is required');
```

Comment:
I agree, consistency would be good. I guess, because these are going to eventually be stripped out for production, we could go with short and sweet. How about simply "xxx is required", i.e., in this case "reviewId is required"?

Reply:
I like "xxx is required"